validator/lib/validateTileset.js

```diff
@@ -32,6 +32,19 @@ function validateNode(root, parent, resolve) {
         var tile = node.node;
         var parent = node.parent;
 
+        if (defined(tile.content.boundingVolume)) {
+            var region = tile.content.boundingVolume.region;
+            var parentRegion = parent.content.boundingVolume.region;
+            for (var i = 0; i < region.length; i++) {
+                if (region[i] > parentRegion[i]) {
+                    return resolve({
+                        result: false,
+                        message: 'Child occupies region greater than parent'
+                    });
+                }
```

Comment:
This check needs to be tweaked. A region is a longitude/latitude rectangle. The test needs to be more like a rectangle inside rectangle test (including also checking the heights). You should be able to use Cesium's `Rectangle` class to help out here. Each region can be converted to a rectangle, and then you can use `Rectangle.contains` on each corner. I would factor out this check into a function rather than in the tree traversal. In terms of contributing back to Cesium, this function may eventually become `Rectangle.containsRectangle`.